content/09.rule8.md

```diff
@@ -0,0 +1 @@
+## Rule 8: A DL model can be, but doesn't need to be a black box
```

Comment:
To be consistent with using a verb/infinitive, how about alternative formulations of this? Off the top of my head sth like this

"Do not necessarily consider a DL model as a black box"